pinot-server/src/main/java/org/apache/pinot/server/starter/helix/SegmentMessageHandlerFactory.java

```diff
@@ -144,6 +147,28 @@ public HelixTaskResult handleMessage()
     }
   }
 
+  private class TableDeletionMessageHandler extends DefaultMessageHandler {
+    TableDeletionMessageHandler(TableDeletionMessage tableDeletionMessage, ServerMetrics metrics,
+        NotificationContext context) {
+      super(tableDeletionMessage, metrics, context);
+    }
+
+    @Override
+    public HelixTaskResult handleMessage()
+        throws InterruptedException {
+      HelixTaskResult helixTaskResult = new HelixTaskResult();
+      _logger.info("Handling message: {}", _message);
```

Comment:
What should this print, won't it just print the hashcode or do we need the content?

Reply:
Good catch, it will print some helix internal states which is not quite informative. Changed to `_logger.info("Handling table deletion message");`